core/block/editor/smartblock/smartblock.go

```diff
@@ -1328,19 +1328,21 @@ func ObjectApplyTemplate(sb SmartBlock, s *state.State, templates ...template.St
 
 func hasChanges(changes []*pb.ChangeContent) bool {
 	for _, ch := range changes {
-		if isStoreOrNotificationChanges(ch) {
+		if isSuitableChanges(ch) {
 			return true
 		}
```

Comment:
It's not clear from the name of the func what hasChanges actually does. It's in the context of the sb package, not notifications

Reply:
We can rename `hasChanges` to something like `hasChangesToPush`, because we can check such changes in that function: store, notifications, devices changes